client/src/actions/team.js

```diff
@@ -26,25 +26,9 @@ export const createTeam = (teamInfo) => async (dispatch) => {
   try{
     await api.createTeam(teamInfo)
     dispatch(successMessageCreator("Team has been created"));
```

Comment:
It isn't too clear why there are two calls made to `dispatch`. Add a comment above each to explain what they're for. E.g. the first dispatch is used to signal that a notification should be shown and the second allows us to know when user has been registered successfully to allow us to go back to the login page.